backend/src/acidwatch_api/models/solubilityccs.py

```diff
@@ -15,14 +15,14 @@ class SolubilityCCSParameters(BaseParameters):
     temperature: float = Parameter(
         15,
         label="Temperature",
-        unit=Unit.CELCIUS,
+        unit=Unit.TEMPERATURE_KELVIN,
         min=-100,
```

Comment:
For Kelvin temperatures cannot be below 0, so make sense to have min 0 and max 400